### ISSUE TITLE
notes are not syncing with iOS 

### ISSUE THREAD

**Issue by GiacomoGiammatteo** [glushchenko/fsnotes#566]
I type a note on my iMac or on my iPad or iPhone and they don't sync.

**Comment by glushchenko** [comment]
Did you have enough space on you device?

**Comment by GiacomoGiammatteo** [comment]
Yes, plenty

**Comment by glushchenko** [comment]
Please open Files.app on iOS and check files availability. FSNotes should be enabled for iCloud Drive usage.

**Comment by sdclee** [comment]
I have just bought he macOS and iOS versions. Files on the Mac are only having the titles synced to IOS whereas a test note on IOS appears on the Mac. I have FSNotes enabled for iCloud on the iPad.

**Comment by glushchenko** [comment]
What is your default storage in Preferences?

<img width="588" alt="image" src="https://user-images.githubusercontent.com/1089917/57543013-dab4a200-735b-11e9-8f89-1253b2ca5488.png">

**Comment by GiacomoGiammatteo** [comment]
Storae is in “iCloud, FS notes”

Jim

**Comment by glushchenko** [comment]
What is in your Files app - FSNotes dir on iOS? Files exist?

**Comment by GiacomoGiammatteo** [comment]
The docs that came with the app, plus one i created

**Comment by glushchenko** [comment]
I don’t think that FSNotes related bug. Looks like iCloud Drive troubles

**Comment by hellonoko** [comment]
I seem to be having this same issue.

I have a folder created on my Mac with a new note... the new note has some text.

On my iPhone the folder and note appeared but are empty...
On my iPad the folder and note appeared but are empty, when I added a image to the note here it shows up only on iPad.

If I look in my iClould files on the mac the image I added on my iPad seems to be there but under the **Archive** folder not the main FSnotes folder.

The same on my iPad.

When I look in my Archive on iPhone or iPad in FSNotes it is empty. However on my Mac it has some notes. This is the archive at the top of FSNotes: NOTES - TODO - ARCHIVE - TRASH not the Archive folder under my iCloud section (MacOS)

I am also noticing now that my iOS apps don't have an iCloud -> Archive folder.

However in my iClould -> FSNotes folder on all platforms I do have a Archive folder....

Sorry if that is not clear!


Moving the files out of the iCloud -> FSNotes -> Archive folder and into the main FSNotes folder seems to haved fixed the problem. Although I had to manually download the info.json text.markdown and text 2.markdown files on iOS platforms and then restart the apps. The image I added on iPad doesn't load up anywhere now as whatever was on macOS seems to have overwritten everything.

**Comment by sdclee** [comment]
Yes I think it is an iCloud issue. I have been experimenting. One or two notes only seem to get the title (and blank content) on the iPad but work fine the other way. Most of the notes sync ok so I'll try deleting the earlier test ones and keep n eye on things. Thanks for the contributions :)

**Comment by cathcartd** [comment]
I am also experiencing what @sdclee is - new notes made on Mac appear on iOS but there’s no content in them. When I go into the files app on iOS and tap on the note to download it and the. Go back to the FSNotes app the content appears.

**Comment by glushchenko** [comment]
@cathcartd what file container and iOS version do you use?

**Comment by GiacomoGiammatteo** [comment]
I have the latest version of both Mac OS and iOS. Apple recommended a reinstall, so I’m going to try that. 

Have a great day,

Giacomo

Giacomo (Jim) Giammatteo is the author of gritty crime dramas <http://giacomogiammatteo.com/> about murder, mystery, and family. And he also writes non-fiction books <http://nomistakes.org/> including the No Mistakes Careers series.

He lives in Texas where he and his wife have an animal sanctuary with 45 loving “friends.”

Book Trailer
 <http://www.youtube.com/watch?v=UL4UVQ0pwys>Website: Giacomo Giammatteo
 <http://giacomogiammatteo.com/>eMail: gg@giacomog.com <mailto:%20gg@giacomog.com>

 <http://www.linkedin.com/in/jimgiammatteo> <http://www.youtube.com/giacomogiammatteo> <https://twitter.com/JimGiammatteo> <http://jgiammatteo.tumblr.com/> <http://www.slideshare.net/JimGiacomoGiammatteo> <http://www.reddit.com/user/Ggiammatteo/> <file:///Users/jgiammatteo/Dropbox/Publishing/Mailing%20lists/mail%20signatures/GG%20signature/www.pinterest.com/jgiammatteo> <http://www.linkedin.com/in/jimgiammatteo> <http://gplus.to/GiacomoGiammatteo> <http://gplus.to/GiacomoGiammatteo> <http://www.facebook.com/GiacomoGiammatteo>
 <http://www.facebook.com/GiacomoGiammatteo>

**Comment by cathcartd** [comment]
@glushchenko - file container on mac app says none, ios version 12.3.1 (fsnotes manager app version 2.6.2), os x os version 10.14.5, fsnotes app 3.3.1 (268)

**Comment by cathcartd** [comment]
My issue seems fixed in iOS fsnotes manager 2.7.0

**Comment by sdclee** [comment]
My files have all disappeared from view in FS Notes 3.36 (272) even though I know they are on the drive (icloud/documents/fsnotes). This is on my Mac 10.14.5.
The weird thing is that on opening snots you see one note, then navigate away to any other folder/tage and everything is blank (see attached screenshots).
My notes are on the iPhone and iPad and my Time Machine backup but I don't want to uninstall/reinstall fs notes and run the risk of losing the notes.
Please advise - thanks.
L
ee

<img width="1700" alt="Screenshot 2019-08-02 at 15 10 10" src="https://user-images.githubusercontent.com/1264229/62376359-604d8580-b538-11e9-989a-bb83ad010b98.png">
<img width="1431" alt="Screenshot 2019-08-02 at 15 10 28" src="https://user-images.githubusercontent.com/1264229/62376361-60e61c00-b538-11e9-9de5-658f050c1ffb.png">
<img width="473" alt="Screenshot 2019-08-02 at 15 11 39" src="https://user-images.githubusercontent.com/1264229/62376373-63e10c80-b538-11e9-8fbd-c238dec54967.png">
<img width="799" alt="Screenshot 2019-08-02 at 15 11 49" src="https://user-images.githubusercontent.com/1264229/62376374-63e10c80-b538-11e9-8994-8f41c55f7efd.png">

**Comment by sdclee** [comment]
Oh Crikey, I just discovered what it was - sorry if I've bothered anyone.
Somewhere along the line I have hit a key combination while not looking at the screen and it has hidden everything. I actually uninstalled and reinstalled it after checking all the notes were on my MacBook Pro first. That made no difference to the mail Mac after reinstalling so I looked at all the menu options:

View/ Hide/show note list was the culprit all along!
